baremetal/platform/cascoda-nuvoton-chili2/port/source/cascoda_chili_secure.c

```diff
@@ -146,8 +147,32 @@ static const IP_T ip_tbl[] = {
 };
 #endif
 
+static uint32_t CHILI_FreqEnumToInt(fsys_mhz aFreqEnum)
```

Comment:
I have a small suggestion for improvement. You can get rid of this function by modifying the enum:
`/****** Enum system core clock frequency [MHz] ******/
typedef enum fsys_mhz
{
	FSYS_4MHZ  = 4000000,
	FSYS_12MHZ = 12000000,
	FSYS_16MHZ = 16000000,
	FSYS_24MHZ = 24000000,
	FSYS_32MHZ = 32000000,
	FSYS_48MHZ = 48000000,
	FSYS_64MHZ = 64000000
} fsys_mhz;`

If I'm not mistaken... But it will increase the memory footprint

Reply:
Good suggestion, applied in b0ed64f. I scaled the numbers to MHz so it still fits in a uint8_t. Doesn't affect ram and saves 12 bytes of code 🎉

Reply:
I checked all works fine. I guess we can close this issue))